assets/images/gallery/expand_icon.svg

```diff
@@ -0,0 +1,14 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="50" height="51" viewBox="0 0 50 51">
+    <defs>
+        <path id="a" d="M9.362 5.643l8.824 9.148a2.852 2.852 0 0 1 0 3.928 2.625 2.625 0 0 1-1.895.814 2.624 2.624 0 0 1-1.894-.813L5.358 9.348v3.77c0 1.535-1.2 2.778-2.679 2.778-1.479 0-2.678-1.243-2.678-2.777V3.09c0-.824.353-1.557.903-2.066C.933.991.957.952.988.919L1 .902l.007-.007c.122-.126.256-.24.4-.34.067-.046.138-.077.207-.116C1.7.39 1.78.339 1.87.3c.086-.037.177-.06.267-.086.078-.024.153-.056.234-.072.165-.034.332-.051.5-.052l.024-.003h9.67c1.479 0 2.678 1.244 2.678 2.778s-1.2 2.778-2.679 2.778H9.362zm5.035 25.838a2.61 2.61 0 0 1 3.788 0 2.85 2.85 0 0 1 0 3.927L9.36 44.56h3.203c1.48 0 2.679 1.244 2.679 2.778s-1.2 2.778-2.679 2.778h-9.67c-.174 0-.35-.02-.523-.056-.08-.015-.155-.047-.233-.07-.09-.029-.18-.05-.267-.087-.092-.04-.176-.093-.263-.143-.066-.038-.135-.067-.198-.112a2.723 2.723 0 0 1-.507-.47A2.802 2.802 0 0 1 0 47.114V37.085c0-1.534 1.2-2.778 2.679-2.778 1.479 0 2.678 1.244 2.678 2.778v3.77l9.04-9.374zm32.894 2.826c1.48 0 2.679 1.244 2.68 2.778v10.028c0 .824-.354 1.556-.903 2.065a2.702 2.702 0 0 1-.508.47c-.06.042-.124.069-.186.105-.091.053-.18.108-.275.15-.085.036-.173.056-.26.083-.081.025-.157.057-.24.073-.172.036-.349.056-.524.056h-9.67c-1.48 0-2.678-1.244-2.678-2.778s1.199-2.778 2.678-2.778h3.204l-8.824-9.15a2.85 2.85 0 0 1 0-3.928 2.61 2.61 0 0 1 3.787 0l9.04 9.375v-3.77c0-1.535 1.2-2.779 2.68-2.779zM33.68 19.45c-.685 0-1.37-.272-1.894-.816a2.85 2.85 0 0 1 0-3.928l8.824-9.15h-3.204c-1.48 0-2.678-1.244-2.678-2.778S35.926 0 37.405 0h9.67c.176 0 .352.019.524.055.082.016.157.048.235.072.09.027.178.048.265.085.096.04.18.096.27.147.063.037.13.066.192.109.189.131.355.293.506.469.55.508.904 1.242.904 2.066V13.03c0 1.535-1.2 2.778-2.679 2.778-1.48 0-2.679-1.243-2.679-2.778v-3.77l-9.04 9.375a2.621 2.621 0 0 1-1.894.814z"/>
+    </defs>
+    <g fill="none" fill-rule="evenodd">
+        <mask id="b" fill="#fff">
```

Comment:
I had never seen this label.
Can you explain to me how it works?

Reply:
This is  an autogenerated file, all our icon should be SVGs downloaded from zeplin.

SVG are vector bases graphics, that is why you can see this code, you can read more about it here:
https://es.wikipedia.org/wiki/Gráficos_vectoriales_escalables